### PR TITLE
McAfee ESM Fixes

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17432

## Description
also fixed to logout method - verified that the session count stayed the same after the test finished.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

### CODE REVIEW

**Comment by yaakovi** [comment]
@kirbles19 
please go over the release notes here:
https://github.com/demisto/content/edit/McAfeeEsmFixes/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3735

File: Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py [hunk 1]
```diff
@@ -904,6 +882,7 @@ def get_minutes_left():
                 triggered_date = alarm['triggeredDate']
                 if next_run is None or next_run < triggered_date:
                     next_run = triggered_date
+                alarm['events'] = esm.list_alarm_events(alarm['ID'])
```
Comment (yuvalbenshalom) [inline Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py]: @yaakovi 
Is it possible that the query for the alarm will return nothing?
If so, will it return just an empty dict without error?

Comment (yaakovi) [reply Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py]: no alarms = empty list
error = raising an exception

**Comment by kirbles19** [comment]
> @kirbles19
> please go over the release notes here:
> https://github.com/demisto/content/edit/McAfeeEsmFixes/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F3735

@yaakovi I don't have access to this link and don't have edit permission on the .yml file.

**Pull request merged**